add-domain

```diff
@@ -236,6 +236,7 @@ sochipark-hotel.ru
 socialearn.co
 soclaiemx.xyz
 spiffcoin.net
+sqbshiexke.ru
```

Comment:
You have added the domain to the wrong list, f I have understood your intention correct.

You like to blacklist this domain as wildcard right? then you need to add to https://github.com/mitchellkrogza/phishing/blob/main/add-wildcard-domain in stead